warlock.trigger.xml

```diff
@@ -518,9 +513,9 @@ flashChatContainerButton(wizContainer, true)</script>
                     <colorTriggerFgColor>#000000</colorTriggerFgColor>
                     <colorTriggerBgColor>#000000</colorTriggerBgColor>
                     <regexCodeList>
-                        <string>^(@) (.+): (.+)</string>
+                        <string>^(\[)(.+): (.+)</string>
                         <string>^&lt;(.+)&gt; (.+): (.+)</string>

```

Comment:
No dobra, ale to też trzeba poprawić.
Mamy więcej linii do rozmów:
line apprentice:
[hh:mm:ss] @ Imie: komunikat
line wiz, mage, ...:
[hh:mm:ss] <LineName> Imie: komunikat

Reply:
No i jeszcze wizline emote, czyli beż dwukropka po imieniu.
U mnie te dwa warianty wyglądają tak:
```
^(\[[^\]]+\]) (@) (\w+)(:)? (.+)
^(\[[^\]]+\]) <(.+)> (\w+)(:)? (.+)
```

No i oczywiście musiałem zmienić indeksy potem w matches odpowiednio:
```
--display(matches)
line = matches[3]
if(line == "@") then
 line = ""
else
 line = "[" .. line .. "] "
end
wizConsole:cecho(matches[2] .. " <light_gray>" .. line .. "<white>" .. matches[4] .. matches[5] .. " " .. matches[6] .. "<reset>\n")
flashChatContainerButton(wizContainer, true)
```